Library/Formula/rbenv.rb

```diff
@@ -1,6 +1,6 @@
 class Rbenv < Formula
-  desc "Ruby environment tool"
-  homepage "https://github.com/rbenv/rbenv"
+  desc "Ruby version manager"
```

Comment:
I don't think it will. I feel like "environment tool" is too ambiguous. By that definition, the dotenv gem for Ruby is also an "environment tool" because it loads values from `.env` file into Ruby environment.

Better suggestions welcome :wink: